sdk/eventhub/Azure.Messaging.EventHubs/README.md

```diff
@@ -146,18 +146,37 @@ For detailed information about these and other exceptions that may occur, please
 
 ## Next steps
 
-Beyond those discussed, the Azure Event Hubs client library offers support for many additional scenarios to help take advantage of the full feature set of the Azure Event Hubs service.  In order to help explore some of these scenarios, the following set of samples is available:
+Beyond the scenarios discussed, the Azure Event Hubs client library offers support for many additional scenarios to help take advantage of the full feature set of the Azure Event Hubs service.  In order to help explore some of these scenarios, the following set of samples is available:
 
-- Hello world
-- Create and Event Hub client with custom options
-- Inspect Event Hub and partition properties
-- Publish events to a specific Event Hub partition
-- Publish events with custom metadata
-- Consume events with as the owner of a partition/consumer group
-- Save the last read event and resume from that point
+- [Hello world](https://raw.githubusercontent.com/Azure/azure-sdk-for-net/master/sdk/eventhub/Azure.Messaging.EventHubs/samples/Sample1_HelloWorld.cs)  
```

Comment:
what is `//raw.githubusercontent.com` ?

Reply:
The content without the Git Hub UI wrapped on it.   Probably not the one that I want here... good catch!    (I probably copy/pasted from the link to the ARM template, which needed to be raw).     I'll fix that up with the next round of changes.